src/mlpack/tests/q_learning_test.cpp

```diff
@@ -438,17 +436,8 @@ BOOST_AUTO_TEST_CASE(DoublePoleCartWithDQN)
       double episodeReturn = agent.Episode();
       episodes += 1;
 
-      if (episodes > 1000)
-      {
-        Log::Debug << "Multiple Pole Cart with DQN failed." << std::endl;
-        converged = false;
-        break;
-      }
-
-      if (episodeReturn >= 380)
+      if (episodeReturn >= 280)
         successive++;
```

Comment:
`successive` may be a little bit misleading of a name now, maybe `episodeSuccesses` or something is better, but it's up to you.